components/batch-system/src/config.rs

```diff
@@ -11,6 +11,7 @@ pub struct Config {
     pub pool_size: usize,
     pub reschedule_duration: ReadableDuration,
     pub low_priority_pool_size: usize,
+    pub before_pause_wait_us: Option<u64>,
```

Comment:
How about reusing `raft_msg_flush_interval_us`?

Reply:
This config is useless for apply batch system. And the `raft_msg_flush_interval_us` belongs to raftstore not batch system so I think just using two configs is ok.
The default value of store batch system's `before_pause_wait_us` is `raft_msg_flush_interval_us `.

Reply:
You can mark it skip serde skip and use `raft_msg_flush_interval_us` to initialize the value.